### PR TITLE
chore: don't merge when the do-not-merge or WIP labels are applied

### PR DESCRIPTION
**Pull request by bvtujo**
Allow a quick way of stopping mergify from auto-merging PRs that need to wait for some reason

---

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .mergify.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ pull_request_rules:
       - -title~=(WIP|wip)
       - -merged
       - -closed
+      - label!=(WIP|do-not-merge)
```
Comment (SoManyHs) [inline .mergify.yml:12]: Just curious, but in this case is there effectively any difference between this and `- label ~=(WIP|do-not-merge)`, as we do [here](https://github.com/aws/amazon-ecs-cli-v2/blob/master/.mergify.yml#L10)?

Comment (SoManyHs) [reply .mergify.yml:12]: (Also noting that we do not currently have either the `WIP` or `do-not-merge` labels)

Comment (bvtujo) [reply .mergify.yml:12]:
```suggestion
      - -label~=(WIP|do-not-merge)
```

**Comment by bvtujo** [comment]
This isn't getting merged because it matches the "WIP in title" condition, so I'm going to merge it manually.

**Pull request merged**